recipes-bsp/u-boot/files/0001-board-raspberrypi-add-serial-and-revision-to-the-dev.patch

```diff
@@ -0,0 +1,82 @@
+From 86cc911aaa958fedf2ea9cb04b4af17f5357815d Mon Sep 17 00:00:00 2001
+From: Anton Gerasimov <anton.gerasimov@here.com>
+Date: Fri, 1 Feb 2019 14:39:48 +0100
+Subject: [PATCH] board: raspberrypi: add serial and revision to the device
+ tree
+
+Raspberry Pi bootloader adds this node to fdt, but if u-boot script
+doesn't reuse the tree provided by it, this information is lost.
+
+Revision and serial are displayed in /proc/cpuinfo after boot.
+
+Signed-off-by: Anton Gerasimov <anton.gerasimov@here.com>
+---
+ board/raspberrypi/rpi/rpi.c | 31 +++++++++++++++++++++++++++++--
+ 1 file changed, 29 insertions(+), 2 deletions(-)
+
+diff --git a/board/raspberrypi/rpi/rpi.c b/board/raspberrypi/rpi/rpi.c
+index 35f5939552..114178397e 100644
+--- a/board/raspberrypi/rpi/rpi.c
++++ b/board/raspberrypi/rpi/rpi.c
+@@ -241,6 +241,8 @@ static uint32_t rev_scheme;
+ static uint32_t rev_type;
+ static const struct rpi_model *model;
+ 
++uint64_t serial;
```

Comment:
static?

Reply:
Oh yes, sure. Unfortunately I've already sent it to the mailing list, maybe they will have more comments.